data/campaigns/Secrets_of_the_Ancients/utils/sota-utils.cfg

```diff
@@ -239,7 +239,7 @@
         name=die
         [filter]
             id=Shynal
-            type=SotA Lich, SotA Ancient Lich, Lich, Ancient Lich
+            type_adv_tree=SotA Lich, SotA Ancient Lich, Lich, Ancient Lich
```

Comment:
The point of the `SotA Lich` is that it can automatically advance to an `SotA Ancient Lich`, so type_adv_tree should be able to omit one of those types.

Reply:
fixed